components/gitpod-protocol/go/gitpod-service.go

```diff
@@ -1481,6 +1487,26 @@ func (gp *APIoverJSONRPC) ResetGenericInvite(ctx context.Context, teamID string)
 	return
 }
 
+func (gp *APIoverJSONRPC) SetTeamMemberRole(ctx context.Context, teamID, userID string, role TeamMemberRole) (err error) {
+	if gp == nil {
+		err = errNotConnected
+		return
+	}
+	_params := []interface{}{teamID, userID, role}
+	err = gp.C.Call(ctx, string(FunctionSetTeamMemberRole), _params, nil)
+	return
+}
+
+func (gp *APIoverJSONRPC) RemoveTeamMember(ctx context.Context, teamID, userID string) (err error) {
+	if gp == nil {
+		err = errNotConnected
+		return
+	}
+	_params := []interface{}{teamID, userID}
+	err = gp.C.Call(ctx, string(FunctionRemoveTeamMember), _params, nil)
+	return
+}
+
```

Comment:
How come this is necessary? `setTeamMemberRole` and `RemoveTeamMember` are already on `server` and presumably were being used, so why do they need to be added here?

Reply:
They were on server, but the Go interface didn't have them. It's not auto-generated, it's hand-coded and there's nothing to ensure the interfaces match :/